CTFd/utils.py

```diff
@@ -575,6 +575,19 @@ def is_port_free(port):
     return True
 
 
+def import_image(name):
+    try:
+        info = json.loads(subprocess.check_output(['docker', 'inspect', '--type=image', name]))
+        print(info)
+        container = Containers(name, "<none>")
```

Comment:
You should be able to simply set the `buildfile` to None here. 

Also can you explicitly specify the keywords here (i.e. `name=name, buildfile=None`)